### PR TITLE
[@types/google-apps-script] Add type for getData request parameter

### PR DESCRIPTION
**Pull request by BertrandDechoux**
See https://developers.google.com/datastudio/connector/reference#getdata

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/datastudio/connector/reference#getdata

### CODE REVIEW

**Review by grant — APPROVED** [review]

File: types/google-apps-script/google-apps-script.data-studio.d.ts [hunk 1]
```diff
@@ -390,6 +390,58 @@ declare namespace GoogleAppsScript {
...
+      lastRefresh: string;
+    }
+      
+    export interface DimensionsFilters {
+      /** The name of the field to be filtered */
+      fieldName: string;
+      /** An array of values to use for the operator. */
+      values: string[];
+      /** "INCLUDE" | "EXCLUDE"	Whether data matching this filter should be included or excluded from the getData() response. */
+      type: string;
```
Comment (amcasey) [inline types/google-apps-script/google-apps-script.data-studio.d.ts]: Any reason not to type it as `"INCLUDE" | "EXCLUDE"`?

**Comment by amcasey** [comment]
Consider adding tests for these new declarations.

**Comment by minestarks** [comment]
Holding off on merging until amcasey's comments are addressed. Thanks

**Review by minestarks — CHANGES_REQUESTED** [review]
Please add a few tests to validate your changes, thanks!

**Comment by BertrandDechoux** [comment]
Still need to add tests.

File: types/google-apps-script/google-apps-script.data-studio.d.ts (hunk 1 above)
Comment (BertrandDechoux) [reply types/google-apps-script/google-apps-script.data-studio.d.ts]: This part is fixed now but I might be missing other tricks from TypeScript.

**Review by amcasey — APPROVED** [review]

**Comment by BertrandDechoux** [comment]
@amcasey Thanks!

**Pull request merged**